docs/proposals/Popover/README.md

```diff
@@ -0,0 +1,131 @@
+...
+const AttachPdfToQuote = (props { checked: boolean}) => {
+	const checkboxRef = useRef()
+	const dontTellMeAnyMore = () => { ... }
+
+	return (
+		<>
+			<Checkbox
+				checked={props.checked}
+				ref={checkboxRef} checked
+			>
+				Attach quote as PDF
+			</Checkbox>
+
+			<Popover
+				dismissable={false} // Rendering my custom dissmiss button
+				attachTo={checkboxRef}
+				arrow={"top-left"}
+				onDismiss={dontTellMeAnyMore}
+			>
+				{renderProps => (
+					{/* Propover contents */}
+					<Card>
+						<strong>
+							Attachments are excluded by default now!
+						</strong>
+						<button onClick={renderProps.dissmiss}>
+							Okay!
+						</button>
+					</Card>
+				)}
+			</Popover>
+		</>
+	)
+}
+```
+
+```tsx
+// OutOfTheBox.tsx
+import { Popover } from '...'}
+
+const OutOfTheBox = () => {
+    return (
+        <PopoverBase
+            attachTo={<button>No scrubs</button>}
+            activateOn={'focus'}
+        >
+            <marquee>Party like it's 1999</marquee>
+        <PopoverBase>
+    )
+}
+
+```
+
+## Props Table
+
+### Popover
+
+| name        | type                                                                           | required? | default   | description                                                                                                                                                                                      |
+| ----------- | ------------------------------------------------------------------------------ | --------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| attachTo    | `HTMLElement \| React.Ref<HTMLElement \| null> \| React.ReactNode `            | ✓         |           | Element the popover will attach to and point at.                                                                                                                                                 |
+| activateOn  | `"click" \| "focus"`                                                           |           | `"click"` | When `attachTo` is React node it it will be cloned and the popover will be toggled _on click_ by default.                                                                                        |
+| dismissable | `boolean`                                                                      |           | `true`    | Whether or not the popover is dismissable.                                                                                                                                                       |
+| toggled     | `boolean`                                                                      |           |           | Control popover viability rather than letting the component take care of it.                                                                                                                     |
```

Comment:
Think that we can simplify this a lot if we look at how modal works with `open` and `onRequestClose`. I'm thinking that especially with this being a pretty foundational component we can keep it pretty barebones then add other triggers into more specific evolutions of this.